lib/sensors/include/URM04Sensor.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+// includes
+#include "BufferedSerial.h"
+#include "Thread.h"
+#include "mbed.h"
+#include "rtos.h"
+
+// constants
+#define urmAccount 1
+#define MAX_TRIES  40
+#define BUAD_RATE  19200
+
+class URM04Sensor {
+ private:
+  // trigger pin
+  DigitalOut _trigPin;
+  // start address
+  uint8_t startAddr;
+  // command buffer
+  uint8_t cmdst[10];
+  // state machine reading step
+  uint8_t readingStep;
+  // arrays needed to read and write distance from sensor
+  int urmID[urmAccount];
+  uint32_t urmData[urmAccount];
+  // timer related members
+  uint64_t managerTimer = 20;
```

Comment:
this is a lot of member variables. if something can just be a local variable it should.